### DISCUSSION TITLE
Web3Auth v4 Release

### DISCUSSION THREAD

**Discussion by shahbaz17** [Web3Auth/Web3Auth#1087] [Announcements]
Hello Web3Auth Devs,

We are excited to share that today we have released `Web3Auth v4.0.0` with some amazing features and bug fixes.

The latest release contains the following changes:
* Sleek Modal UI with all new design elements.
* Web3Auth is faster than ever, with Lazy loading support added to Modal.
* `French` - `fr` is now officially supported on Web3Auth Modal and Openlogin Screens. More language support to come in upcoming releases. Wish to add a new language? Check out https://github.com/Web3Auth/web3auth-locales and submit your changes as a Pull Request.
* With this release, there are some breaking changes too. Please look at the following and update before upgrading to `v4`.
  * The Base Adapter interface has changed.
    * `sessionTime`, `chainConfig`, and `web3AuthNetwork` is added.
  * `setAdapterSettings()` function is introduced to help you change adapter settings.
  * The `uiConfig` in `@web3auth/modal` natively has a new parameter `modalZIndex`, helping you set a custom z-index value for your implementation.
* `Wallet Connect v1 is sunsetting by Dec 31, 2022`. **Wallet Connect v2 is coming soon.**
* A new variable to showcase the connected adapter name is introduced. `web3auth.connectedAdapterName`
* `@web3auth/single-factor-auth` now no longer takes a network parameter in `init()`. It’s been moved to the constructor.

We can't wait for you all to try the latest SDK, and we'll be looking forward to your feedback.

**Comment by yashovardhan** [comment]
![AA410739-09D1-4502-9432-BABEFA3F57A7](https://user-images.githubusercontent.com/21066442/208151836-c970ce3d-c548-4b37-a48f-ed3c93a39b92.png)
![360F252B-2110-40B8-AB88-25A74AD51379](https://user-images.githubusercontent.com/21066442/208151847-4bf6a613-5e2c-463f-b1a5-53953d51b282.png)

**Comment by Animalmix55** [comment]
@shahbaz17 Any updates on WalletConnect V2 support?
